src/site/content/en/blog/building-a-breadcrumbs-component/index.md

```diff
@@ -0,0 +1,498 @@
+---
+layout: post
+title: Building a breadcrumbs component
+subhead: A foundational overview of how to build a responsive and accessible breadcrumbs component for users to traverse your experience
+authors:
+  - adamargyle
+description: A foundational overview of how to build a responsive and accessible breadcrumbs component for users to traverse your experience
+date: 2021-07-14
+thumbnail: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/kCeb8N9YlMYsI5O5ChC9.png
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/dk9VreVtYiF3LBkRpZlg.png
+tags:
+  - blog
+  - css
+  - dom
+  - javascript
+  - interaction
```

Comment:
```suggestion
```

Reply:
This tag is invalid, you can find a list of valid tags here: 
https://github.com/GoogleChrome/web.dev/blob/main/src/site/_data/tagsData.json

Feel free to file an issue and a PR if you think a new tag should be added to the list.